### DISCUSSION TITLE
TechEmpower benchmarks

### DISCUSSION THREAD

**Discussion by mrMiiao** [sparckles/Robyn#334] [General]
Why don't you try participating in the TechEmpower benchmarks?

**Comment by sansyrox** [comment]
Hey @mrMiiao ,

We have already submitted in TechEmpowered. See here https://www.techempower.com/benchmarks/#section=test&runid=43469a25-670e-4bda-a3c3-652839435784&test=plaintext&l=hra0hr-35n&w=18y6o-0-0-0

**Comment by cirospaciari** [comment]
@mrMiiao robyn is in the TechEmPower check https://tfb-status.techempower.com/ to see the most recent.

The most recent and complete is in:
https://www.techempower.com/benchmarks/#section=test&runid=124fd2af-3030-4315-8876-e1db1fa91193&test=plaintext&l=hra0hr-35r

The next will be in:
https://www.techempower.com/benchmarks/#section=test&runid=9bb6e0fb-1619-447c-b5f4-572207cfbc72&test=plaintext&l=hra0hr-35r

Is funny to see frameworks getting better (or not) with new runs and also this is used by Microsoft team and PHP team to find some performance issues.
PHP JIT: https://github.com/TechEmpower/FrameworkBenchmarks/issues/7703
asp.net core: https://twitter.com/davidfowl/status/1592346242855886848

**Reply by mrMiiao** [reply]
That’s great

**Comment by shaohaiyang** [comment]
Why don't you try participating in the https://web-frameworks-benchmark.netlify.app/

**Reply by sansyrox** [reply]
@shaohaiyang 👋 

Because I just learned about these benchmarks today 😅 . Would you like to open an issue on Robyn or maybe PR on web-frameworks benchmark repo? I will be happy to take it forward 😄

**Reply by yonnig** [reply]
In JSON serialization results of Round 23, Robyn is below a pure Python ASGI framework as Starlette, and far from Japronto which heavily relies on C. What can explain this?
